helper-cli/src/main/kotlin/commands/ExportPathExcludesCommand.kt

```diff
@@ -88,10 +91,15 @@ internal class ExportPathExcludesCommand : CommandWithHelp() {
 /**
  * Serialize this [RepositoryPathExcludes] to the given [targetFile] as YAML.
  */
-internal fun RepositoryPathExcludes.writeAsYaml(targetFile: File) =
+internal fun RepositoryPathExcludes.writeAsYaml(targetFile: File) {
+    targetFile.parentFile.apply {
```

Comment:
Please use `targetFile.parentFile.apply { safeMkdirs() }`.

Reply:
oh, sorry. Didn't mean to push this. But will remember that hint ;-)